osu.Game.Rulesets.Osu/Scoring/OsuPerformanceCalculator.cs

```diff
@@ -32,7 +32,7 @@ public OsuPerformanceCalculator(Ruleset ruleset, IBeatmap beatmap, Score score)
             countHitCircles = Beatmap.HitObjects.Count(h => h is HitCircle);
 
             beatmapMaxCombo = Beatmap.HitObjects.Count();
-            beatmapMaxCombo += Beatmap.HitObjects.OfType<Slider>().Sum(s => s.NestedHitObjects.Count) + 1;
+            beatmapMaxCombo += Beatmap.HitObjects.OfType<Slider>().Count();
```

Comment:
this doesn't look right... let's double-check the corresponding code in `osu-stable`/`osu-performance`.